src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -16,7 +16,6 @@ export type Organization = {
   projects: Project[];
   access: string[];
   features: string[];
-  teams: Team[];
```

Comment:
This looks not right and could impact other parts of the app. I remember seeing a LightWeightOrganization in one of @davidenwang 's changes. That seems like a more suitable type for your use case.

Reply:
I added this incorrectly I believe. I think david actually fixed it.